test/org/zaproxy/zap/extension/ScannerTestUtils.java

```diff
@@ -133,7 +133,7 @@ public void describeTo(Description description) {
                 description.appendText("alert name ").appendValue(Constant.messages.getString(key));
             }
 
-            @Override
+            //@Override
```

Comment:
I don't exactly know what is going on here, but why comment a line instead of just deleting it?

Reply:
The problem is that the compiler is picking an older version in which the class does not have this method. It will be uncommented once the libraries are "fixed".